lib/symboldatabase.cpp

```diff
@@ -5149,14 +5149,23 @@ const Type* SymbolDatabase::findVariableType(const Scope *start, const Token *ty
     return nullptr;
 }
 
+static bool hasEmptyCaptureList(const Token* tok) {
+    if (!Token::simpleMatch(tok, "{"))
+        return false;
+    const Token* listTok = tok->astParent();
+    if (Token::simpleMatch(listTok, "("))
+        listTok = listTok->astParent();
+    return Token::Match(listTok, "[ ]");
```

Comment:
I guess our internal checker will require simpleMatch here.

Reply:
Looks like a FN in the checker.